hyperspy/roi.py

```diff
@@ -1040,6 +1040,50 @@ def length(self):
         d_row, d_col = p1 - p0
         return np.hypot(d_row, d_col)
 
+    def angle(self, axis='horizontal', units='degrees'):
+        """"Angle between ROI line and selected axis
+
+        Parameters
+        ----------
+        axis : str, {'horizontal', 'vertical', 'x', 'y'}, optional
+            Select axis with respect to which teh angle to be measured.
+            'x' is alias to 'horizontal' and 'y' is 'vertical'
+            (Default: 'horizontal')
+        units : str, {'degrees', 'radians'}
+            Select units in which the angle to be calculated
+            (Default: 'degrees')
+
+        Returns
+        -------
+        angle : float
+            Angle in radians or degrees in range [-pi, pi]
```

Comment:
typo: "degrees in range [-pi, pi]", actually this is already documented above, so this is a bit redundant.